test/e2e/verify.integration.test.js

```diff
@@ -51,19 +57,27 @@ describe("verify", () => {
       .expect(res => {
         expect(res.body).toEqual({
           hash: {
-            valid: true
+            checksumMatch: true
           },
           issued: {
-            valid: false,
-            issued: {
-              "0x007d40224f6562461633ccfbaffd359ebb2fc9ba": false
-            }
+            issuedOnAll: false,
+            details: [
+              {
+                address: "0x007d40224f6562461633ccfbaffd359ebb2fc9ba",
+                error:
+                  'call exception (address="0x007d40224f6562461633ccfbaffd359ebb2fc9ba", method="isIssued(bytes32)", args=["0x1a040999254caaf7a33cba67ec6a9b862da1dacf8a0d1e3bb76347060fc615d6"], version=4.0.37)',
```

Comment:
This might throw, you might want to use a matcher instead. The version number changes.